avocado/utils/path.py

```diff
@@ -184,8 +187,9 @@ def usable_ro_dir(directory):
             pass
     else:
         try:
-            init_dir(directory)
-            return True
+            if create:
+                init_dir(directory)
+                return True
```

Comment:
What do you think use the create parameter in the `else` changing it to `elif`? It keeps the try block little bit smaller, with just the necessary code inside it.
```python
    if os.path.isdir(directory):
        try:
            os.chdir(directory)
            os.chdir(cwd)
            return True
        except OSError:
            pass
    elif create:
        try:
            init_dir(directory)
            return True
        except OSError:
            pass
```

Reply:
Good point!